packages/editor/src/dataviews/types.ts

```diff
@@ -14,6 +14,19 @@ export interface BasePost {
 	type: string;
 	id: string | number;
 	blocks?: Object[];
+
+	// Not sure if these should be in template, template part or pattern types
```

Comment:
Probably not (at least most of them - not sure about `meta` or `links`). Maybe `BasePost` should become `BaseEntity` and create another one `Post` to add these props? 🤔

Reply:
I improved this in the last commit [f687689](https://github.com/WordPress/gutenberg/pull/64441/commits/f687689879035581a3556acab9397cf429a6db68)